gunicorn_config 2.py

```diff
@@ -0,0 +1,9 @@
+from os import environ
```

Comment:
I'm not sure what this second file is for - is this intentional?

Reply:
Yeah... no idea why there are two. I'll erase it and see if something breaks.

Reply:
Didn't cause any issues, file has been deleted